code/game/objects/items/__item.dm

```diff
@@ -378,7 +378,7 @@
 	return ..(user, distance, "", jointext(desc_comp, "<br/>"))
 
 /obj/item/check_mousedrop_adjacency(var/atom/over, var/mob/user)
-	. = (loc == user && istype(over, /obj/screen/inventory)) || ..()
```

Comment:
```
/obj/item/organ/internal/augment/active/cyberbrain/handle_mouse_drop(atom/over, mob/user, params)
	if(!istype(over, /obj/screen))
		attack_self(user)
		return TRUE
	. = ..()
```
dunno, but it's not the only place that checks for specifically screen as opposed to inventory. was just going for what current code did, since by checking both with `||` we're defaulting to the less-specific one